modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/generators/SW360UpdaterImpl.java

```diff
@@ -49,19 +53,23 @@ public SW360UpdaterImpl(SW360MetaDataUpdater sw360MetaDataUpdater,
                         .map(SW360License::getShortName)
                         .collect(Collectors.toSet());
             }
-
-            final SW360Release sw360ReleaseFromArtifact = SW360ReleaseAdapterUtils.convertToRelease(artifact);
-            sw360ReleaseFromArtifact.setMainLicenseIds(licenseIds);
-            SW360Release sw360ReleaseFinal = sw360MetaDataUpdater.getOrCreateRelease(sw360ReleaseFromArtifact);
-            if (sw360MetaDataUpdater.isUploadSources()
-                    && sw360ReleaseFinal.get_Links().getSelf() != null
-                    && !sw360ReleaseFinal.get_Links().getSelf().getHref().isEmpty()) {
-                Map<Path, SW360AttachmentType> attachments = SW360AttachmentAdapterUtils.getAttachmentsFromArtifact(artifact);
-                if (!attachments.isEmpty()) {
-                    sw360ReleaseFinal = sw360MetaDataUpdater.uploadAttachments(sw360ReleaseFinal, attachments);
+            try {
+                final SW360Release sw360ReleaseFromArtifact = SW360ReleaseAdapterUtils.convertToRelease(artifact);
+                sw360ReleaseFromArtifact.setMainLicenseIds(licenseIds);
+                SW360Release sw360ReleaseFinal = sw360MetaDataUpdater.getOrCreateRelease(sw360ReleaseFromArtifact);
+                if (sw360MetaDataUpdater.isUploadSources()
+                        && sw360ReleaseFinal.get_Links().getSelf() != null
+                        && !sw360ReleaseFinal.get_Links().getSelf().getHref().isEmpty()) {
+                    Map<Path, SW360AttachmentType> attachments = SW360AttachmentAdapterUtils.getAttachmentsFromArtifact(artifact);
+                    if (!attachments.isEmpty()) {
+                        sw360ReleaseFinal = sw360MetaDataUpdater.uploadAttachments(sw360ReleaseFinal, attachments);
+                    }
                 }
+                releases.add(sw360ReleaseFinal);
+            } catch(Exception e) {
+                LOGGER.warn("Release for {} will not be created in SW360 due to missing facts. Reason: {}",
```

Comment:
Also log the exception as debug entry! Actually I think this should even be logged as an error because the user might ignore warnings, but with this issue here, he will never get an approval for the project, he has to do something and this is necessary to address with him

Actually, the log message should contain a hint about what needs to be done. Without the component in SW360, the compliance office will not work on the component. What can he do to solve this? Good question, isn't it, can he do something in the config file?

I have to admit, I miss the situation as of now, where this case can happen at all because we will detect either a coordinate because we get information from a package manager, or we detect a file so we have at least a filename. Is there any way, in which we can detect a dependency without any information? I have no idea how. Since we do not expect that, perhaps it is even feasible to break completely, so no caught exception here. If something happens, that cannot be, it is best to not react at all, so that the user is coming to us in order to solve the issue.

Reply:
Yes, when a component is added via the antennaconf.xml this can happen. 
Especially then it is important that the user adds enough information so this component can be used correctly.

Reply:
THen we should break at reading in the config file if we cannot detect enough informationen for a coordinate.

Reply:
Follow up issue for config.xml https://github.com/eclipse/antenna/issues/384